wpilib/wpilib/analogtriggeroutput.py

```diff
@@ -45,6 +45,12 @@ class AnalogTriggerOutput(DigitalSource):
     Using the averaging engine may help with this, but rotational speeds of the
     sensor will then be limited.
     """
+    class AnalogTriggerType:
+        """Defines the state in which the :class:`.AnalogTrigger` triggers"""
```

Comment:
is the only point of this class decl to insert a docstring?

Reply:
No it gets used in other classes

Reply:
seems elsewhere I've seen eg
```
AnalogTriggerType = hal.AnalogTriggerType
```

Reply:
look at `counter.py`

Reply:
I personally have no idea why it'd be here rather than AnalogTrigger, but guess this is the status quo.